src/algorithms.jl

```diff
@@ -37,6 +37,9 @@ struct Heun <: OrdinaryDiffEqAdaptiveAlgorithm end
 struct Ralston <: OrdinaryDiffEqAdaptiveAlgorithm end
 struct Midpoint <: OrdinaryDiffEqAdaptiveAlgorithm end
 struct RK4 <: OrdinaryDiffEqAdaptiveAlgorithm end
+struct Anas5{T} <: OrdinaryDiffEqAlgorithm
+ w::T		
+end
```

Comment:
`w` is not a keyword argument in here. You need to write an outer constructor.
```julia
Anas5(; w=1) = Anas5(w)
```

Reply:
Why would 1 be a good default? I don't think it should have a default so the current setup is good.